python/paddle/distributed/fleet/base/fleet_base.py

```diff
@@ -579,27 +597,25 @@ def distributed_optimizer(self, optimizer, strategy=None):
         """
         self.user_defined_optimizer = optimizer
 
-        if strategy == None:
-            strategy = DistributedStrategy()
+        if strategy is not None:
```

Comment:
update the doc in distributed_optimizer?

Reply:
done